Build Errors with CI/CD Pipeline

After the version update and theme update of Hugo, we are seeing this build error on the CI/CD pipeline that deploys the site to S3

```
2025-09-17T05:25:03.7796326Z Current runner version: '2.328.0'
2025-09-17T05:25:03.7818796Z ##[group]Runner Image Provisioner
2025-09-17T05:25:03.7819979Z Hosted Compute Agent
2025-09-17T05:25:03.7820638Z Version: 20250829.383
2025-09-17T05:25:03.7821279Z Commit: 27cb235aab5b0e52e153a26cd86b4742e89dac5d
2025-09-17T05:25:03.7821965Z Build Date: 2025-08-29T13:48:48Z
2025-09-17T05:25:03.7822649Z ##[endgroup]
2025-09-17T05:25:03.7823274Z ##[group]Operating System
2025-09-17T05:25:03.7823865Z Ubuntu
2025-09-17T05:25:03.7824337Z 24.04.3
2025-09-17T05:25:03.7824818Z LTS
2025-09-17T05:25:03.7825289Z ##[endgroup]
2025-09-17T05:25:03.7825791Z ##[group]Runner Image
2025-09-17T05:25:03.7826396Z Image: ubuntu-24.04
2025-09-17T05:25:03.7826907Z Version: 20250907.24.1
2025-09-17T05:25:03.7827979Z Included Software: https://github.com/actions/runner-images/blob/ubuntu24/20250907.24/images/ubuntu/Ubuntu2404-Readme.md
2025-09-17T05:25:03.7829808Z Image Release: https://github.com/actions/runner-images/releases/tag/ubuntu24%2F20250907.24
2025-09-17T05:25:03.7830965Z ##[endgroup]
2025-09-17T05:25:03.7833422Z ##[group]GITHUB_TOKEN Permissions
2025-09-17T05:25:03.7835687Z Actions: write
2025-09-17T05:25:03.7836267Z Attestations: write
2025-09-17T05:25:03.7836762Z Checks: write
2025-09-17T05:25:03.7837374Z Contents: write
2025-09-17T05:25:03.7837932Z Deployments: write
2025-09-17T05:25:03.7838453Z Discussions: write
2025-09-17T05:25:03.7839121Z Issues: write
2025-09-17T05:25:03.7839860Z Metadata: read
2025-09-17T05:25:03.7840418Z Models: read
2025-09-17T05:25:03.7840974Z Packages: write
2025-09-17T05:25:03.7841529Z Pages: write
2025-09-17T05:25:03.7842071Z PullRequests: write
2025-09-17T05:25:03.7842645Z RepositoryProjects: write
2025-09-17T05:25:03.7843498Z SecurityEvents: write
2025-09-17T05:25:03.7844248Z Statuses: write
2025-09-17T05:25:03.7844808Z ##[endgroup]
2025-09-17T05:25:03.7847180Z Secret source: Actions
2025-09-17T05:25:03.7848250Z Prepare workflow directory
2025-09-17T05:25:03.8157331Z Prepare all required actions
2025-09-17T05:25:03.8194652Z Getting action download info
2025-09-17T05:25:04.1207766Z Download action repository 'actions/checkout@v2' (SHA:ee0669bd1cc54295c223e0bb666b733df41de1c5)
2025-09-17T05:25:04.5663997Z Download action repository 'peaceiris/actions-hugo@v2' (SHA:16361eb4acea8698b220b76c0d4e84e1fd22c61d)
2025-09-17T05:25:04.7881992Z Download action repository 'aws-actions/configure-aws-credentials@v1' (SHA:67fbcbb121271f7775d2e7715933280b06314838)
2025-09-17T05:25:05.0783908Z Complete job name: build-and-deploy
2025-09-17T05:25:05.1462633Z ##[group]Run actions/checkout@v2
2025-09-17T05:25:05.1463518Z with:
2025-09-17T05:25:05.1463911Z   submodules: true
2025-09-17T05:25:05.1464395Z   repository: lmcconnell1665/McConnellWeb
2025-09-17T05:25:05.1465150Z   token: ***
2025-09-17T05:25:05.1465545Z   ssh-strict: true
2025-09-17T05:25:05.1465985Z   persist-credentials: true
2025-09-17T05:25:05.1466468Z   clean: true
2025-09-17T05:25:05.1466864Z   fetch-depth: 1
2025-09-17T05:25:05.1467283Z   lfs: false
2025-09-17T05:25:05.1467698Z   set-safe-directory: true
2025-09-17T05:25:05.1468421Z ##[endgroup]
2025-09-17T05:25:05.2485011Z Syncing repository: lmcconnell1665/McConnellWeb
2025-09-17T05:25:05.2486951Z ##[group]Getting Git version info
2025-09-17T05:25:05.2487887Z Working directory is '/home/runner/work/McConnellWeb/McConnellWeb'
2025-09-17T05:25:05.2488969Z [command]/usr/bin/git version
2025-09-17T05:25:05.2542859Z git version 2.51.0
2025-09-17T05:25:05.2567340Z ##[endgroup]
2025-09-17T05:25:05.2581947Z Temporarily overriding HOME='/home/runner/work/_temp/b04eba8e-939e-443d-a3a0-16cb5290b439' before making global git config changes
2025-09-17T05:25:05.2583493Z Adding repository directory to the temporary git global config as a safe directory
2025-09-17T05:25:05.2585226Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/McConnellWeb/McConnellWeb
2025-09-17T05:25:05.2620192Z Deleting the contents of '/home/runner/work/McConnellWeb/McConnellWeb'
2025-09-17T05:25:05.2624074Z ##[group]Initializing the repository
2025-09-17T05:25:05.2627206Z [command]/usr/bin/git init /home/runner/work/McConnellWeb/McConnellWeb
2025-09-17T05:25:05.2723700Z hint: Using 'master' as the name for the initial branch. This default branch name
2025-09-17T05:25:05.2724869Z hint: is subject to change. To configure the initial branch name to use in all
2025-09-17T05:25:05.2726292Z hint: of your new repositories, which will suppress this warning, call:
2025-09-17T05:25:05.2727610Z hint:
2025-09-17T05:25:05.2728543Z hint: 	git config --global init.defaultBranch <name>
2025-09-17T05:25:05.2729797Z hint:
2025-09-17T05:25:05.2730788Z hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
2025-09-17T05:25:05.2731912Z hint: 'development'. The just-created branch can be renamed via this command:
2025-09-17T05:25:05.2732704Z hint:
2025-09-17T05:25:05.2733120Z hint: 	git branch -m <name>
2025-09-17T05:25:05.2733603Z hint:
2025-09-17T05:25:05.2734252Z hint: Disable this message with "git config set advice.defaultBranchName false"
2025-09-17T05:25:05.2735458Z Initialized empty Git repository in /home/runner/work/McConnellWeb/McConnellWeb/.git/
2025-09-17T05:25:05.2738569Z [command]/usr/bin/git remote add origin https://github.com/lmcconnell1665/McConnellWeb
2025-09-17T05:25:05.2773026Z ##[endgroup]
2025-09-17T05:25:05.2773888Z ##[group]Disabling automatic garbage collection
2025-09-17T05:25:05.2776083Z [command]/usr/bin/git config --local gc.auto 0
2025-09-17T05:25:05.2803451Z ##[endgroup]
2025-09-17T05:25:05.2804335Z ##[group]Setting up auth
2025-09-17T05:25:05.2809086Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2025-09-17T05:25:05.2837354Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :"
2025-09-17T05:25:05.3250654Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2025-09-17T05:25:05.3278066Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :"
2025-09-17T05:25:05.3493397Z [command]/usr/bin/git config --local http.https://github.com/.extraheader AUTHORIZATION: basic ***
2025-09-17T05:25:05.3526423Z ##[endgroup]
2025-09-17T05:25:05.3527270Z ##[group]Fetching the repository
2025-09-17T05:25:05.3534144Z [command]/usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +5e5b1c01c6379e38ec3f7812be1477413898942c:refs/remotes/origin/master
2025-09-17T05:25:05.5059536Z remote: Enumerating objects: 96, done.        
2025-09-17T05:25:05.5062525Z remote: Counting objects:   1% (1/96)        
2025-09-17T05:25:05.5064363Z remote: Counting objects:   2% (2/96)        
2025-09-17T05:25:05.5069103Z remote: Counting objects:   3% (3/96)        
2025-09-17T05:25:05.5071280Z remote: Counting objects:   4% (4/96)        
2025-09-17T05:25:05.5073008Z remote: Counting objects:   5% (5/96)        
2025-09-17T05:25:05.5074286Z remote: Counting objects:   6% (6/96)        
2025-09-17T05:25:05.5075208Z remote: Counting objects:   7% (7/96)        
2025-09-17T05:25:05.5076109Z remote: Counting objects:   8% (8/96)        
2025-09-17T05:25:05.5076996Z remote: Counting objects:   9% (9/96)        
2025-09-17T05:25:05.5077935Z remote: Counting objects:  10% (10/96)        
2025-09-17T05:25:05.5078923Z remote: Counting objects:  11% (11/96)        
2025-09-17T05:25:05.5080212Z remote: Counting objects:  12% (12/96)        
2025-09-17T05:25:05.5081235Z remote: Counting objects:  13% (13/96)        
2025-09-17T05:25:05.5082289Z remote: Counting objects:  14% (14/96)        
2025-09-17T05:25:05.5083319Z remote: Counting objects:  15% (15/96)        
2025-09-17T05:25:05.5084375Z remote: Counting objects:  16% (16/96)        
2025-09-17T05:25:05.5085470Z remote: Counting objects:  17% (17/96)        
2025-09-17T05:25:05.5086543Z remote: Counting objects:  18% (18/96)        
2025-09-17T05:25:05.5087879Z remote: Counting objects:  19% (19/96)        
2025-09-17T05:25:05.5088954Z remote: Counting objects:  20% (20/96)        
2025-09-17T05:25:05.5090250Z remote: Counting objects:  21% (21/96)        
2025-09-17T05:25:05.5091311Z remote: Counting objects:  22% (22/96)        
2025-09-17T05:25:05.5092379Z remote: Counting objects:  23% (23/96)        
2025-09-17T05:25:05.5093577Z remote: Counting objects:  25% (24/96)        
2025-09-17T05:25:05.5094982Z remote: Counting objects:  26% (25/96)        
2025-09-17T05:25:05.5096376Z remote: Counting objects:  27% (26/96)        
2025-09-17T05:25:05.5097763Z remote: Counting objects:  28% (27/96)        
2025-09-17T05:25:05.5099329Z remote: Counting objects:  29% (28/96)        
2025-09-17T05:25:05.5100813Z remote: Counting objects:  30% (29/96)        
2025-09-17T05:25:05.5102232Z remote: Counting objects:  31% (30/96)        
2025-09-17T05:25:05.5103627Z remote: Counting objects:  32% (31/96)        
2025-09-17T05:25:05.5105040Z remote: Counting objects:  33% (32/96)        
2025-09-17T05:25:05.5106459Z remote: Counting objects:  34% (33/96)        
2025-09-17T05:25:05.5107922Z remote: Counting objects:  35% (34/96)        
2025-09-17T05:25:05.5109540Z remote: Counting objects:  36% (35/96)        
2025-09-17T05:25:05.5111020Z remote: Counting objects:  37% (36/96)        
2025-09-17T05:25:05.5112498Z remote: Counting objects:  38% (37/96)        
2025-09-17T05:25:05.5113954Z remote: Counting objects:  39% (38/96)        
2025-09-17T05:25:05.5115407Z remote: Counting objects:  40% (39/96)        
2025-09-17T05:25:05.5116889Z remote: Counting objects:  41% (40/96)        
2025-09-17T05:25:05.5118445Z remote: Counting objects:  42% (41/96)        
2025-09-17T05:25:05.5120200Z remote: Counting objects:  43% (42/96)        
2025-09-17T05:25:05.5121755Z remote: Counting objects:  44% (43/96)        
2025-09-17T05:25:05.5123310Z remote: Counting objects:  45% (44/96)        
2025-09-17T05:25:05.5124857Z remote: Counting objects:  46% (45/96)        
2025-09-17T05:25:05.5126430Z remote: Counting objects:  47% (46/96)        
2025-09-17T05:25:05.5127999Z remote: Counting objects:  48% (47/96)        
2025-09-17T05:25:05.5129734Z remote: Counting objects:  50% (48/96)        
2025-09-17T05:25:05.5131259Z remote: Counting objects:  51% (49/96)        
2025-09-17T05:25:05.5132720Z remote: Counting objects:  52% (50/96)        
2025-09-17T05:25:05.5134174Z remote: Counting objects:  53% (51/96)        
2025-09-17T05:25:05.5135808Z remote: Counting objects:  54% (52/96)        
2025-09-17T05:25:05.5137301Z remote: Counting objects:  55% (53/96)        
2025-09-17T05:25:05.5138755Z remote: Counting objects:  56% (54/96)        
2025-09-17T05:25:05.5140369Z remote: Counting objects:  57% (55/96)        
2025-09-17T05:25:05.5141799Z remote: Counting objects:  58% (56/96)        
2025-09-17T05:25:05.5143228Z remote: Counting objects:  59% (57/96)        
2025-09-17T05:25:05.5144668Z remote: Counting objects:  60% (58/96)        
2025-09-17T05:25:05.5146095Z remote: Counting objects:  61% (59/96)        
2025-09-17T05:25:05.5147513Z remote: Counting objects:  62% (60/96)        
2025-09-17T05:25:05.5148971Z remote: Counting objects:  63% (61/96)        
2025-09-17T05:25:05.5150566Z remote: Counting objects:  64% (62/96)        
2025-09-17T05:25:05.5151988Z remote: Counting objects:  65% (63/96)        
2025-09-17T05:25:05.5153427Z remote: Counting objects:  66% (64/96)        
2025-09-17T05:25:05.5154871Z remote: Counting objects:  67% (65/96)        
2025-09-17T05:25:05.5156333Z remote: Counting objects:  68% (66/96)        
2025-09-17T05:25:05.5157781Z remote: Counting objects:  69% (67/96)        
2025-09-17T05:25:05.5159449Z remote: Counting objects:  70% (68/96)        
2025-09-17T05:25:05.5160916Z remote: Counting objects:  71% (69/96)        
2025-09-17T05:25:05.5162342Z remote: Counting objects:  72% (70/96)        
2025-09-17T05:25:05.5163756Z remote: Counting objects:  73% (71/96)        
2025-09-17T05:25:05.5165174Z remote: Counting objects:  75% (72/96)        
2025-09-17T05:25:05.5166753Z remote: Counting objects:  76% (73/96)        
2025-09-17T05:25:05.5168175Z remote: Counting objects:  77% (74/96)        
2025-09-17T05:25:05.5169813Z remote: Counting objects:  78% (75/96)        
2025-09-17T05:25:05.5171271Z remote: Counting objects:  79% (76/96)        
2025-09-17T05:25:05.5172715Z remote: Counting objects:  80% (77/96)        
2025-09-17T05:25:05.5174135Z remote: Counting objects:  81% (78/96)        
2025-09-17T05:25:05.5175577Z remote: Counting objects:  82% (79/96)        
2025-09-17T05:25:05.5176994Z remote: Counting objects:  83% (80/96)        
2025-09-17T05:25:05.5178412Z remote: Counting objects:  84% (81/96)        
2025-09-17T05:25:05.5179940Z remote: Counting objects:  85% (82/96)        
2025-09-17T05:25:05.5181353Z remote: Counting objects:  86% (83/96)        
2025-09-17T05:25:05.5182764Z remote: Counting objects:  87% (84/96)        
2025-09-17T05:25:05.5184164Z remote: Counting objects:  88% (85/96)        
2025-09-17T05:25:05.5185575Z remote: Counting objects:  89% (86/96)        
2025-09-17T05:25:05.5186969Z remote: Counting objects:  90% (87/96)        
2025-09-17T05:25:05.5188363Z remote: Counting objects:  91% (88/96)        
2025-09-17T05:25:05.5189864Z remote: Counting objects:  92% (89/96)        
2025-09-17T05:25:05.5191261Z remote: Counting objects:  93% (90/96)        
2025-09-17T05:25:05.5192668Z remote: Counting objects:  94% (91/96)        
2025-09-17T05:25:05.5194086Z remote: Counting objects:  95% (92/96)        
2025-09-17T05:25:05.5195494Z remote: Counting objects:  96% (93/96)        
2025-09-17T05:25:05.5196898Z remote: Counting objects:  97% (94/96)        
2025-09-17T05:25:05.5198305Z remote: Counting objects:  98% (95/96)        
2025-09-17T05:25:05.5199826Z remote: Counting objects: 100% (96/96)        
2025-09-17T05:25:05.5201272Z remote: Counting objects: 100% (96/96), done.        
2025-09-17T05:25:05.5202748Z remote: Compressing objects:   1% (1/86)        
2025-09-17T05:25:05.5204121Z remote: Compressing objects:   2% (2/86)        
2025-09-17T05:25:05.5205501Z remote: Compressing objects:   3% (3/86)        
2025-09-17T05:25:05.5206915Z remote: Compressing objects:   4% (4/86)        
2025-09-17T05:25:05.5208296Z remote: Compressing objects:   5% (5/86)        
2025-09-17T05:25:05.5209763Z remote: Compressing objects:   6% (6/86)        
2025-09-17T05:25:05.5211142Z remote: Compressing objects:   8% (7/86)        
2025-09-17T05:25:05.5212649Z remote: Compressing objects:   9% (8/86)        
2025-09-17T05:25:05.5214047Z remote: Compressing objects:  10% (9/86)        
2025-09-17T05:25:05.5215404Z remote: Compressing objects:  11% (10/86)        
2025-09-17T05:25:05.5216832Z remote: Compressing objects:  12% (11/86)        
2025-09-17T05:25:05.5218217Z remote: Compressing objects:  13% (12/86)        
2025-09-17T05:25:05.5219683Z remote: Compressing objects:  15% (13/86)        
2025-09-17T05:25:05.5221067Z remote: Compressing objects:  16% (14/86)        
2025-09-17T05:25:05.5222448Z remote: Compressing objects:  17% (15/86)        
2025-09-17T05:25:05.5223828Z remote: Compressing objects:  18% (16/86)        
2025-09-17T05:25:05.5225201Z remote: Compressing objects:  19% (17/86)        
2025-09-17T05:25:05.5226560Z remote: Compressing objects:  20% (18/86)        
2025-09-17T05:25:05.5411931Z remote: Compressing objects:  22% (19/86)        
2025-09-17T05:25:05.5561066Z remote: Compressing objects:  23% (20/86)        
2025-09-17T05:25:05.5682470Z remote: Compressing objects:  24% (21/86)        
2025-09-17T05:25:05.5762544Z remote: Compressing objects:  25% (22/86)        
2025-09-17T05:25:05.5764424Z remote: Compressing objects:  26% (23/86)        
2025-09-17T05:25:05.5786833Z remote: Compressing objects:  27% (24/86)        
2025-09-17T05:25:05.5788343Z remote: Compressing objects:  29% (25/86)        
2025-09-17T05:25:05.5790024Z remote: Compressing objects:  30% (26/86)        
2025-09-17T05:25:05.5791445Z remote: Compressing objects:  31% (27/86)        
2025-09-17T05:25:05.5792863Z remote: Compressing objects:  32% (28/86)        
2025-09-17T05:25:05.5907909Z remote: Compressing objects:  33% (29/86)        
2025-09-17T05:25:05.5911117Z remote: Compressing objects:  34% (30/86)        
2025-09-17T05:25:05.5914332Z remote: Compressing objects:  36% (31/86)        
2025-09-17T05:25:05.5916156Z remote: Compressing objects:  37% (32/86)        
2025-09-17T05:25:05.5917678Z remote: Compressing objects:  38% (33/86)        
2025-09-17T05:25:05.5919345Z remote: Compressing objects:  39% (34/86)        
2025-09-17T05:25:05.5920884Z remote: Compressing objects:  40% (35/86)        
2025-09-17T05:25:05.5922371Z remote: Compressing objects:  41% (36/86)        
2025-09-17T05:25:05.5923857Z remote: Compressing objects:  43% (37/86)        
2025-09-17T05:25:05.5925256Z remote: Compressing objects:  44% (38/86)        
2025-09-17T05:25:05.5926628Z remote: Compressing objects:  45% (39/86)        
2025-09-17T05:25:05.5927984Z remote: Compressing objects:  46% (40/86)        
2025-09-17T05:25:05.5929450Z remote: Compressing objects:  47% (41/86)        
2025-09-17T05:25:05.5930821Z remote: Compressing objects:  48% (42/86)        
2025-09-17T05:25:05.5932169Z remote: Compressing objects:  50% (43/86)        
2025-09-17T05:25:05.5997637Z remote: Compressing objects:  51% (44/86)        
2025-09-17T05:25:05.6002035Z remote: Compressing objects:  52% (45/86)        
2025-09-17T05:25:05.6005298Z remote: Compressing objects:  53% (46/86)        
2025-09-17T05:25:05.6008191Z remote: Compressing objects:  54% (47/86)        
2025-09-17T05:25:05.6010953Z remote: Compressing objects:  55% (48/86)        
2025-09-17T05:25:05.6013231Z remote: Compressing objects:  56% (49/86)        
2025-09-17T05:25:05.6014622Z remote: Compressing objects:  58% (50/86)        
2025-09-17T05:25:05.6016476Z remote: Compressing objects:  59% (51/86)        
2025-09-17T05:25:05.6018662Z remote: Compressing objects:  60% (52/86)        
2025-09-17T05:25:05.6020567Z remote: Compressing objects:  61% (53/86)        
2025-09-17T05:25:05.6022094Z remote: Compressing objects:  62% (54/86)        
2025-09-17T05:25:05.6023954Z remote: Compressing objects:  63% (55/86)        
2025-09-17T05:25:05.6026215Z remote: Compressing objects:  65% (56/86)        
2025-09-17T05:25:05.6027941Z remote: Compressing objects:  66% (57/86)        
2025-09-17T05:25:05.6030551Z remote: Compressing objects:  67% (58/86)        
2025-09-17T05:25:05.6032126Z remote: Compressing objects:  68% (59/86)        
2025-09-17T05:25:05.6033715Z remote: Compressing objects:  69% (60/86)        
2025-09-17T05:25:05.6035706Z remote: Compressing objects:  70% (61/86)        
2025-09-17T05:25:05.6037732Z remote: Compressing objects:  72% (62/86)        
2025-09-17T05:25:05.6040367Z remote: Compressing objects:  73% (63/86)        
2025-09-17T05:25:05.6042281Z remote: Compressing objects:  74% (64/86)        
2025-09-17T05:25:05.6043641Z remote: Compressing objects:  75% (65/86)        
2025-09-17T05:25:05.6045014Z remote: Compressing objects:  76% (66/86)        
2025-09-17T05:25:05.6046410Z remote: Compressing objects:  77% (67/86)        
2025-09-17T05:25:05.6047738Z remote: Compressing objects:  79% (68/86)        
2025-09-17T05:25:05.6049781Z remote: Compressing objects:  80% (69/86)        
2025-09-17T05:25:05.6051128Z remote: Compressing objects:  81% (70/86)        
2025-09-17T05:25:05.6052805Z remote: Compressing objects:  82% (71/86)        
2025-09-17T05:25:05.6054508Z remote: Compressing objects:  83% (72/86)        
2025-09-17T05:25:05.6056096Z remote: Compressing objects:  84% (73/86)        
2025-09-17T05:25:05.6057840Z remote: Compressing objects:  86% (74/86)        
2025-09-17T05:25:05.6059358Z remote: Compressing objects:  87% (75/86)        
2025-09-17T05:25:05.6060731Z remote: Compressing objects:  88% (76/86)        
2025-09-17T05:25:05.6062067Z remote: Compressing objects:  89% (77/86)        
2025-09-17T05:25:05.6063855Z remote: Compressing objects:  90% (78/86)        
2025-09-17T05:25:05.6065448Z remote: Compressing objects:  91% (79/86)        
2025-09-17T05:25:05.6066991Z remote: Compressing objects:  93% (80/86)        
2025-09-17T05:25:05.6068321Z remote: Compressing objects:  94% (81/86)        
2025-09-17T05:25:05.6070553Z remote: Compressing objects:  95% (82/86)        
2025-09-17T05:25:05.6072382Z remote: Compressing objects:  96% (83/86)        
2025-09-17T05:25:05.6073820Z remote: Compressing objects:  97% (84/86)        
2025-09-17T05:25:05.6075153Z remote: Compressing objects:  98% (85/86)        
2025-09-17T05:25:05.6076500Z remote: Compressing objects: 100% (86/86)        
2025-09-17T05:25:05.6077932Z remote: Compressing objects: 100% (86/86), done.        
2025-09-17T05:25:06.0677329Z remote: Total 96 (delta 1), reused 69 (delta 0), pack-reused 0 (from 0)        
2025-09-17T05:25:06.0815241Z From https://github.com/lmcconnell1665/McConnellWeb
2025-09-17T05:25:06.0817076Z  * [new ref]         5e5b1c01c6379e38ec3f7812be1477413898942c -> origin/master
2025-09-17T05:25:06.0847653Z ##[endgroup]
2025-09-17T05:25:06.0850226Z ##[group]Determining the checkout info
2025-09-17T05:25:06.0852858Z ##[endgroup]
2025-09-17T05:25:06.0854989Z ##[group]Checking out the ref
2025-09-17T05:25:06.0857563Z [command]/usr/bin/git checkout --progress --force -B master refs/remotes/origin/master
2025-09-17T05:25:06.1527766Z Reset branch 'master'
2025-09-17T05:25:06.1529737Z branch 'master' set up to track 'origin/master'.
2025-09-17T05:25:06.1532604Z ##[endgroup]
2025-09-17T05:25:06.1534979Z ##[group]Setting up auth for fetching submodules
2025-09-17T05:25:06.1538326Z [command]/usr/bin/git config --global http.https://github.com/.extraheader AUTHORIZATION: basic ***
2025-09-17T05:25:06.1583398Z [command]/usr/bin/git config --global --unset-all url.https://github.com/.insteadOf
2025-09-17T05:25:06.1615053Z [command]/usr/bin/git config --global --add url.https://github.com/.insteadOf git@github.com:
2025-09-17T05:25:06.1645912Z [command]/usr/bin/git config --global --add url.https://github.com/.insteadOf org-56060247@github.com:
2025-09-17T05:25:06.1673516Z ##[endgroup]
2025-09-17T05:25:06.1674943Z ##[group]Fetching submodules
2025-09-17T05:25:06.1678363Z [command]/usr/bin/git submodule sync
2025-09-17T05:25:06.1915540Z [command]/usr/bin/git -c protocol.version=2 submodule update --init --force --depth=1
2025-09-17T05:25:06.2142113Z Submodule 'themes/LoveIt' (https://github.com/dillonzq/LoveIt.git) registered for path 'themes/LoveIt'
2025-09-17T05:25:06.2172309Z Cloning into '/home/runner/work/McConnellWeb/McConnellWeb/themes/LoveIt'...
2025-09-17T05:25:08.5487377Z From https://github.com/dillonzq/LoveIt
2025-09-17T05:25:08.5488171Z  * branch            f59fb4ecc2ce6c95e0b3395e3d3da9637d5581cd -> FETCH_HEAD
2025-09-17T05:25:08.8019136Z Submodule path 'themes/LoveIt': checked out 'f59fb4ecc2ce6c95e0b3395e3d3da9637d5581cd'
2025-09-17T05:25:08.8031726Z [command]/usr/bin/git submodule foreach git config --local gc.auto 0
2025-09-17T05:25:08.8237265Z Entering 'themes/LoveIt'
2025-09-17T05:25:08.8267177Z ##[endgroup]
2025-09-17T05:25:08.8267990Z ##[group]Persisting credentials for submodules
2025-09-17T05:25:08.8272723Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'url\.https\:\/\/github\.com\/\.insteadOf' && git config --local --unset-all 'url.https://github.com/.insteadOf' || :"
2025-09-17T05:25:08.8477157Z Entering 'themes/LoveIt'
2025-09-17T05:25:08.8531876Z [command]/usr/bin/git submodule foreach sh -c "git config --local 'http.https://github.com/.extraheader' 'AUTHORIZATION: basic ***' && git config --local --show-origin --name-only --get-regexp remote.origin.url"
2025-09-17T05:25:08.8734588Z Entering 'themes/LoveIt'
2025-09-17T05:25:08.8771933Z file:/home/runner/work/McConnellWeb/McConnellWeb/.git/modules/themes/LoveIt/config	remote.origin.url
2025-09-17T05:25:08.8793977Z [command]/usr/bin/git submodule foreach git config --local --add 'url.https://github.com/.insteadOf' 'git@github.com:'
2025-09-17T05:25:08.8999748Z Entering 'themes/LoveIt'
2025-09-17T05:25:08.9031509Z [command]/usr/bin/git submodule foreach git config --local --add 'url.https://github.com/.insteadOf' 'org-56060247@github.com:'
2025-09-17T05:25:08.9239728Z Entering 'themes/LoveIt'
2025-09-17T05:25:08.9268659Z ##[endgroup]
2025-09-17T05:25:08.9301696Z [command]/usr/bin/git log -1 --format='%H'
2025-09-17T05:25:08.9324005Z '5e5b1c01c6379e38ec3f7812be1477413898942c'
2025-09-17T05:25:08.9511440Z ##[group]Run peaceiris/actions-hugo@v2
2025-09-17T05:25:08.9511734Z with:
2025-09-17T05:25:08.9511917Z   hugo-version: 0.72.0
2025-09-17T05:25:08.9512109Z   extended: false
2025-09-17T05:25:08.9512311Z ##[endgroup]
2025-09-17T05:25:08.9930836Z Hugo version: 0.72.0
2025-09-17T05:25:09.2490435Z [command]/usr/bin/tar xz --warning=no-unknown-keyword --overwrite -C /home/runner/actions_hugo/_temp -f /home/runner/work/_temp/59fbfb8d-9c7c-4a14-8087-70602dc85fd7
2025-09-17T05:25:09.5188718Z [command]/home/runner/actions_hugo/bin/hugo version
2025-09-17T05:25:09.5376589Z Hugo Static Site Generator v0.72.0-8A7EF3CF linux/amd64 BuildDate: 2020-05-31T12:07:45Z
2025-09-17T05:25:09.5564638Z ##[group]Run make clean
2025-09-17T05:25:09.5564966Z [36;1mmake clean[0m
2025-09-17T05:25:09.5565221Z [36;1mhugo[0m
2025-09-17T05:25:09.5620562Z shell: /usr/bin/bash -e {0}
2025-09-17T05:25:09.5620824Z ##[endgroup]
2025-09-17T05:25:09.5923142Z echo "deleting generated HTML"
2025-09-17T05:25:09.5929342Z deleting generated HTML
2025-09-17T05:25:09.5930442Z rm -rf public
2025-09-17T05:25:09.7260073Z Building sites … ERROR 2025/09/17 05:25:09 render of "page" failed: execute of template failed: template: _default/single.html:24:16: executing "_default/single.html" at <partial "header.html" .>: error calling partial: "/home/runner/work/McConnellWeb/McConnellWeb/themes/LoveIt/layouts/partials/header.html:60:23": execute of template failed: template: partials/header.html:60:23: executing "partials/header.html" at <hugo>: can't evaluate field IsMultilingual in type interface {}
2025-09-17T05:25:09.7263139Z ERROR 2025/09/17 05:25:09 render of "page" failed: execute of template failed: template: _default/single.html:24:16: executing "_default/single.html" at <partial "header.html" .>: error calling partial: "/home/runner/work/McConnellWeb/McConnellWeb/themes/LoveIt/layouts/partials/header.html:60:23": execute of template failed: template: partials/header.html:60:23: executing "partials/header.html" at <hugo>: can't evaluate field IsMultilingual in type interface {}
2025-09-17T05:25:09.7265910Z ERROR 2025/09/17 05:25:09 render of "page" failed: execute of template failed: template: posts/single.html:24:16: executing "posts/single.html" at <partial "header.html" .>: error calling partial: "/home/runner/work/McConnellWeb/McConnellWeb/themes/LoveIt/layouts/partials/header.html:60:23": execute of template failed: template: partials/header.html:60:23: executing "partials/header.html" at <hugo>: can't evaluate field IsMultilingual in type interface {}
2025-09-17T05:25:09.7268658Z ERROR 2025/09/17 05:25:09 render of "page" failed: execute of template failed: template: posts/single.html:24:16: executing "posts/single.html" at <partial "header.html" .>: error calling partial: "/home/runner/work/McConnellWeb/McConnellWeb/themes/LoveIt/layouts/partials/header.html:60:23": execute of template failed: template: partials/header.html:60:23: executing "partials/header.html" at <hugo>: can't evaluate field IsMultilingual in type interface {}
2025-09-17T05:25:09.7271669Z Error: Error building site: failed to render pages: render of "page" failed: execute of template failed: template: _default/single.html:24:16: executing "_default/single.html" at <partial "header.html" .>: error calling partial: "/home/runner/work/McConnellWeb/McConnellWeb/themes/LoveIt/layouts/partials/header.html:60:23": execute of template failed: template: partials/header.html:60:23: executing "partials/header.html" at <hugo>: can't evaluate field IsMultilingual in type interface {}
2025-09-17T05:25:09.7273284Z Total in 112 ms
2025-09-17T05:25:09.7294570Z ##[error]Process completed with exit code 255.
2025-09-17T05:25:09.7381481Z Post job cleanup.
2025-09-17T05:25:09.8232308Z [command]/usr/bin/git version
2025-09-17T05:25:09.8267836Z git version 2.51.0
2025-09-17T05:25:09.8312297Z Temporarily overriding HOME='/home/runner/work/_temp/966f3ca4-da19-4fff-94de-1d92c5e8123e' before making global git config changes
2025-09-17T05:25:09.8313593Z Adding repository directory to the temporary git global config as a safe directory
2025-09-17T05:25:09.8316774Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/McConnellWeb/McConnellWeb
2025-09-17T05:25:09.8348414Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2025-09-17T05:25:09.8380413Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :"
2025-09-17T05:25:09.8589648Z Entering 'themes/LoveIt'
2025-09-17T05:25:09.8645901Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2025-09-17T05:25:09.8666429Z http.https://github.com/.extraheader
2025-09-17T05:25:09.8676965Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2025-09-17T05:25:09.8706438Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :"
2025-09-17T05:25:09.8907518Z Entering 'themes/LoveIt'
2025-09-17T05:25:09.8930921Z http.https://github.com/.extraheader
2025-09-17T05:25:09.9092670Z Cleaning up orphan processes
```

--- a/.github/workflows/build-and-deploy.yml
+++ b/.github/workflows/build-and-deploy.yml
@@ -14,7 +14,7 @@ jobs:
     
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
         with:
           submodules: true
         
@@ -22,7 +22,8 @@ jobs:
       - name: Setup-hugo
         uses: peaceiris/actions-hugo@v2
         with:
-          hugo-version: '0.72.0'
+          hugo-version: 'latest'
+          extended: true
           
       # Builds the hugo website
       - name: Build-site
@@ -32,7 +33,7 @@ jobs:
           
       # Grants action AWS permissions for s3 bucket
       - name: Configure-AWS-credentials
-        uses: aws-actions/configure-aws-credentials@v1
+        uses: aws-actions/configure-aws-credentials@v4
         with:
           aws-access-key-id: ${{ secrets.MCCONNELL_WEB_AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.MCCONNELL_WEB_AWS_SECRET_ACCESS_KEY }}

